### PR TITLE
Feature/ben/board data react

### PR DESCRIPTION
**Pull request by benyitzhaki**

### CODE REVIEW

File: apps/board-data-react/src/App.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+  changedNewItemBoard = event => {
+    this.setState({ newItemBoard: event.target.value });
+  };
+
+  render() {
+    const { boards } = this.state;
+
+    let boardOptions = boards.map(board => (
+      <option value={board.id} key={`sb-${board.id}`}>
```
Comment (itaycohen) [inline apps/board-data-react/src/App.js]: what is "sb" stand for?

File: apps/board-data-react/src/App.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  };
+
+  changedNewItemBoard = event => {
+    this.setState({ newItemBoard: event.target.value });
+  };
+
+  render() {
+    const { boards } = this.state;
+
+    let boardOptions = boards.map(board => (
```
Comment (itaycohen) [inline apps/board-data-react/src/App.js]: can be in different method
renderBoardsOptions()

File: apps/board-data-react/src/App.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+                      board.items.map(item => (
+                        <li key={`item-${item.id}`}>{item.name}</li>
+                      ))}
+                  </ul>
+                </li>
+              </ul>
+            ))}
+        </div>
+
+        <form onSubmit={this.addNewItem} className="add-item">
```
Comment (itaycohen) [inline apps/board-data-react/src/App.js]: Nice!

File: apps/board-data-react/src/App.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+          <input
```
Comment (itaycohen) [inline apps/board-data-react/src/App.js]: can be also in different method

File: apps/board-data-react/src/App.js [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+            type="text"
+            placeholder=""
+            onChange={this.changedNewItemName}
+          ></input>
+          <select onChange={this.changedNewItemBoard}>
+            <option name="selected-board-id" id="empty-board">
+              select board
```
Comment (itaycohen) [inline apps/board-data-react/src/App.js]: Select board

File: apps/board-data-react/src/App.js (hunk 1 above)
Comment (benyitzhaki) [reply apps/board-data-react/src/App.js]: renamed :)

Comment (benyitzhaki) [reply apps/board-data-react/src/App.js]: changed it to use keys (was using id by mistake at the beginning, thats why the prefix was added)

**Pull request merged**